pycolmap/scene/image.h

```diff
@@ -286,19 +287,5 @@ void BindImage(py::module& m) {
       .def("__copy__", [](const Image& self) { return Image(self); })
       .def("__deepcopy__",
            [](const Image& self, py::dict) { return Image(self); })
-      .def("__repr__", [](const Image& self) { return PrintImage(self); })
-      .def("summary", [](const Image& self) {
-        std::stringstream ss;
-        ss << "Image:\n\timage_id = "
-           << (self.ImageId() != kInvalidImageId
-                   ? std::to_string(self.ImageId())
-                   : "Invalid")
-           << "\n\tcamera_id = "
-           << (self.HasCamera() ? std::to_string(self.CameraId()) : "Invalid")
-           << "\n\tname = " << self.Name()
-           << "\n\ttriangulated = " << self.NumPoints3D() << "/"
-           << self.NumPoints2D() << "\n\tcam_from_world = ["
-           << self.CamFromWorld().ToMatrix() << "]";
-        return ss.str();
-      });
+      .def("__repr__", &PrintImage);
```

Comment:
I think there should be a way to print all the parameters of an image, including the pose. I suggest to either bring the summary back (my preferred option), or add the pose to `__repr__`.

Reply:
Made Image a dataclass to automatically generate the summary function.

Reply:
Great!